src/style/blocks/input.css

```diff
@@ -0,0 +1,46 @@
+@import '../variables.css';
+
+.search-form__input {
+  height: 100%;
+  width: 100%;
+  border: none;
+}
+
+.search-form__input::placeholder {
+  color: var(--text-color);
+}
+
+.search-form--large .search-form__input {
+  font-size: 16px;
+}
+
+.search-form--small .search-form__input {
+  font-size: 14px;
+  font-weight: 400;
```

Comment:
you can create a generic class for input and modifiers.
always use onle 1 class, otherwise it will confuse you and make your code less readable